CHANGELOG.md

```diff
@@ -1,10 +1,48 @@
 # Changelog
 
-## main / unreleased
-
-* [BUGFIX] OTLP receiver: Generate `target_info` samples between the earliest and latest samples per resource. #16737
-* [BUGFIX] Config: Infer escaping scheme when scrape config validation scheme is set.
+## 3.7.0-rc.0 / 2025-10-02
+
```

Comment:
we can have `[CHANGE] Remote-write: Deprecate `prometheus_remote_storage_{samples,exemplars,histograms}_in_total` and `prometheus_remote_storage_highest_timestamp_in_seconds` metrics, see their respective descriptions for alternatives.`
from https://github.com/prometheus/prometheus/pull/17065 at the top here.